content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -493,7 +493,7 @@ spec:
           port:
             type: string
   conversion:
-    # a Webhook strategy instruct API server to call an external webhook for any conversion between custom resources.
+    # the Webhook strategy instructs the API server to call an external webhook for any conversion between custom resources.
```

Comment:
This doesn't have to be "the webhook strategy", IMO.

Reply:
- What are the conversions between? For example, if I have a FooBar named `thing1` and a different FooBar named `thing2`, is that a conversion (no)
   - it's a conversion between storage serializations, right?
- Optionally, mention what other options there are apart from `Webhook`